src/dashboard/home-dashboard.component.tsx

```diff
@@ -5,26 +5,26 @@ import { UserHasAccessReact } from "@openmrs/esm-api";
 
 export default function HomeDashboard(props: HomeDashboardProps) {
   return (
-    <div className={styles.homeDashboard}>
-      <section className={styles.mainSection}>
-        <UserHasAccessReact privilege="View Patients">
```

Comment:
It looks like we accidentally removed UserHasAccessReact. Was that on purpose? If not, let's bring it back -- we need it.

Reply:
Not on purpose, let me get to it.

Reply:
Changes updated, please review.